miso-service/src/main/java/uk/ac/bbsrc/tgac/miso/service/impl/DefaultProjectService.java

```diff
@@ -203,6 +203,7 @@ private void applyChanges(Project original, Project project) {
     original.setSamplesExpected(project.getSamplesExpected());
     original.setContact(project.getContact());
     original.setAdditionalDetails(project.getAdditionalDetails());
+    original.setDeliverableId(project.getDeliverableId());
```

Comment:
Load the `Deliverable` in `loadChildEntities` (after changing the field from ID to `Deliverable`)